engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js

```diff
@@ -169,5 +188,11 @@ angular.module('Bastion.repositories').controller('NewRepositoryController',
                 return OSVersions.osVersionsParam($scope.repository.os_versions);
             };
 
+            $scope.fixRemoteOptionName = function(str) {
```

Comment:
I'm going to remove this and add a 'title' key to each generic_remote_option. I don't think it's good to assume that the name of the parameter will make the best title in the UI in every case.